package.json

```diff
@@ -39,6 +39,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^11.0.4",
     "@testing-library/user-event": "^7.2.1",
+    "jest-environment-jsdom": "^26.5.2",
     "jest-environment-jsdom-sixteen": "^1.0.3"
```

Comment:
As Nathan mentioned this problem we had was listen on the lesson plan for unit testing